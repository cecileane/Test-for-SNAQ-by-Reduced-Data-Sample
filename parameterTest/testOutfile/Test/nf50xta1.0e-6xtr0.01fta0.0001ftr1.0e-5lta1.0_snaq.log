optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.0001,
                       xtolAbs=1.0e-6, xtolRel=0.01.
 max number of failed proposals = 50, liktolAbs = 1.0.
rootname for files: nf50xta1.0e-6xtr0.01fta0.0001ftr1.0e-5lta1.0_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 60399
seed: 60399 for run 1
Sat Oct  1 17:18:12 2016

 BEGIN SNaQ for run 1, seed 60399 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 78 (failures: 50) with -loglik=215.7344 and ht_min=[0.42251,2.21138,0.0,0.11872,1.00295]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 6		 6		 1	 |	 NA 		 NA 		 NA 
mvorigin	 25		 25		 0	 |	 0		 0		 --
mvtarget	 17		 17		 1	 |	 0		 0		 --
chdir		 6		 6		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 24		 16		 3	 |	 0		 0		 --
Total		 78		 70		 6	 |	 0		 0		 0
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 0.0		 0.0

 FINISHED SNaQ for run 1, -loglik of best 215.73440079950086
(((1,(6,(#H7:::0.42250756664394823,5):2.2113831487447215):1.0029483613434975):0.11871940341379653,(3)#H7:::0.5774924333560518):0.0,4,2);
---------------------
seed: 48094 for run 2
Sat Oct  1 17:18:16 2016

 BEGIN SNaQ for run 2, seed 48094 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 61 (failures: 50) with -loglik=126.74251 and ht_min=[0.2856,0.48942,0.51916,10.0,1.51213]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 15		 15		 0	 |	 0		 0		 --
mvtarget	 20		 20		 0	 |	 0		 0		 --
chdir		 6		 6		 2	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 19		 17		 1	 |	 1		 1		 --
Total		 61		 59		 4	 |	 1		 1		 1
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 2, -loglik of best 126.74250981975555
((3,(5,(6,#H7:::0.2856030467401321):10.0):0.5191608425341439):0.4894189936371129,(2,(1)#H7:::0.714396953259868):1.5121333324942252,4);
---------------------

Sat Oct  1 17:18:21 2016
MaxNet is ((3,(5,(6,#H7:::0.2856030467401321):10.0):0.5191608425341439):0.4894189936371129,(2,(1)#H7:::0.714396953259868):1.5121333324942252,4); 
with -loglik 126.74250981975555
