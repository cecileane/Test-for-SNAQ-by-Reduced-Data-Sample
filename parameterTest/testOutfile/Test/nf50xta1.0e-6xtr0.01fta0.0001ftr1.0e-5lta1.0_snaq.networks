((3,(5,(6,#H7:::0.2856030467401321):10.0):0.5191608425341439):0.4894189936371129,(2,(1)#H7:::0.714396953259868):1.5121333324942252,4);, with -loglik 126.74250981975555 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
((3,(5,(6)#H7:::0.6926775540510743):1.7275641911363702):0.4617022657542153,(2,(1,#H7:::0.3073224459489257):10.0):0.6630084300824601,4);, with -loglik 128.22847215326257
((4)#H7:::0.6214074089501767,3,(5,(6,(1,(#H7:::0.3785925910498233,2):9.913029724613581):1.155603698238655):0.21624686883261313):0.03145113336587427);, with -loglik 171.96860240016474
((3,(5,(6,(1,(2)#H7:::0.6010522442774547):2.9413697624141086):0.15504669906616522):0.024706773420723124):2.0679870754526006,#H7:::0.39894775572254526,4);, with -loglik 192.9771847562525
((3)#H7:::0.5894560014461585,(2,(1,((5,#H7:::0.4105439985538415):3.4099536282995095,6):1.0105109240746812):0.08999835580975014):0.08073663427196584,4);, with -loglik 214.82813334291328
((3,(5)#H7:::0.6144539776912944):1.9568797262360047,(2,(1,(#H7:::0.3855460223087056,6):9.912520052212464):8.959318365062634):0.36811339920026803,4);, with -loglik 2010.4192855414813
