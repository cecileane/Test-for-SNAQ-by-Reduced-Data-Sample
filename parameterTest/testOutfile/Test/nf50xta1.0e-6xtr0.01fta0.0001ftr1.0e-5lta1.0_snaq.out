((3,(5,(6,#H7:::0.2856030467401321):10.0):0.5191608425341439):0.4894189936371129,(2,(1)#H7:::0.714396953259868):1.5121333324942252,4); -Ploglik = 126.74250981975555
 Dendroscope: ((3,(5,(6,#H7):10.0):0.5191608425341439):0.4894189936371129,(2,(1)#H7):1.5121333324942252,4);
 Elapsed time: 9.287477301 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 ((3,(5,(6,#H7:::0.2856030467401321):10.0):0.5191608425341439):0.4894189936371129,(2,(1)#H7:::0.714396953259868):1.5121333324942252,4);, with -loglik 126.74250981975555
 (((1,(6,(#H7:::0.42250756664394823,5):2.2113831487447215):1.0029483613434975):0.11871940341379653,(3)#H7:::0.5774924333560518):0.0,4,2);, with -loglik 215.73440079950086
-------
